lib/checks/hasUnsupportedNodes.js

```diff
@@ -1,30 +1,29 @@
-const Node = require('../node');
+const {query} = require('math-nodes');
+
 const resolvesToConstant = require('./resolvesToConstant');
 
 function hasUnsupportedNodes(node) {
-  if (Node.Type.isParenthesis(node)) {
```

Comment:
I messed up.  I should add this back in.  We do still have parenthesis nodes, but they probably won't be that useful.  Things like `(1 + 2) + 3` are represented as:
```
{ 
   type: 'Apply',
   op: 'add',
   args: [
      { type: 'Apply', op: 'add', args: [ 1, 2 ] },  // full structure omitted
      3,
   ]
}
```
`math-parser`'s `print` function will output `(1 + 2) + 3` when passed this AST.  The only time a `paren` node is require is when it's presences can't otherwise be deduced from the structure of the AST.  One example would be `1 + (x)`.